source/renderer/app/api/api.js

```diff
@@ -1123,12 +1124,17 @@ const _createWalletFromServerData = action(
       balance.total.unit === 'lovelace'
         ? new BigNumber(balance.total.quantity).dividedBy(LOVELACES_PER_ADA)
         : new BigNumber(balance.total.quantity);
+    const walletRewardAmount =
```

Comment:
@yakovkaravelov please use the same solution here as we do for the balance: https://github.com/input-output-hk/daedalus/pull/1643/files#diff-6aa9e91ae277a5d21b8e1bb0b03ffafbR1124
This rewards amount should be in a big number type.